docs/content/dagster-cloud/managing-deployments/alerts/managing-alerts-in-ui.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: "Managing alert policies in Dagster Cloud | Dagster Docs"
+description: "Create and manage alert policies using the Dagster UI."
+
+platform_type: "cloud"
+feature: "alerting"
+---
+
+# Managing alert policies in Dagster Cloud
+
+<ApplicableDagsterProduct />
+
+In this guide, we'll walk you through managing alert policies in Dagster Cloud.
+
+**Prefer to work in code?** You can also define and set alert policies using the [`dagster-cloud` CLI](/dagster-cloud/managing-deployments/alerts/managing-alerts-cli).
+
+---
+
+## Prerequisites
+
+To follow the steps in this guide, you'll need:
+
+- **Organization, Admin, or Editor permissions in Dagster Cloud**, which are required to create and manage alerts
+- **Optional**: If using an integration like [Microsoft Teams](/dagster-cloud/managing-deployments/alerts/microsoft-teams) or [Slack](/dagster-cloud/managing-deployments/alerts/slack) as a notification service, complete the integration setup before proceeding. Refer to the [Supported notification services](/dagster-cloud/managing-deployments/alerts#supported-notification-services) for a complete list of integrations.
+
+---
+
+## Creating alert policies
+
+1. Sign in to your Dagster Cloud account.
+
+2. In the top navigation, click **Deployment**.
+
+3. Click the **Alerts** tab.
+
+4. Click **+ Create alert policy**.
+
+5. In the **Create alert policy** window, select an [alert type](/dagster-cloud/managing-deployments/alerts#alert-policy-types) and enter a name and description for the alert.
+
+6. **If creating an asset or run alert**, fill in the following:
+
+   <details>
+     <summary>
+       <strong>Asset alert policies</strong>
```

Comment:
I think we should remove this section. It only covers 2 of the alert types, and duplicates the info in the alert policy table. We could link back there